docs/Env_Setup.rst

```diff
@@ -108,6 +108,188 @@ with a minimal fedora 23 installation. It will leave any logs under the `logs`
 directory and any generated artifacts under `exported-artifacts`.
 
 
-
    .. _`official yapf repo`: https://github.com/google/yapf
    .. _`copr`: https://copr.Fedoraproject.org/coprs/fulltext/?fulltext=yapf
+
+
+Running Lago in virtualenv
+---------------------------
+This was tested on a fresh Fedora-24 Cloud image, but should work on Fedora 23
+and most likely with little package-names resolving on CentOS >7.
+
+Install dependencies::
+
+  > sudo dnf install -y redhat-rpm-config libvirt libvirt-devel \
+     libguestfs-devel libguestfs-tools qemu-img qemu-kvm \
+     libvirt-daemon git gcc libffi-devel python-devel \
+     openssl-devel bats python-magic python-libguestfs
+
+Note that ``python-magic`` and ``python-libguestfs`` are currently not
```

Comment:
I made a mistake: currently, `file-magic` pip package [0.3.0](https://pypi.python.org/pypi/file-magic/) contains https://github.com/file/file/commit/8048d19f40df4e7a2175a231fa525bc202d6b006 (a commit between 5.25 and 5.26) not 5.28. Due to a regression, this version isn't compatible with Python 2, the bug has been fixed in [file 5.26](https://github.com/file/file/commit/5560418ebc6f94e80953c8646383fdb70758fc7a). Using lago, the bug is encountered when `template_store` directory is populated.

`file` project provides [Python binding](https://github.com/file/file/blob/master/python/). `file-magic` pip package contains `magic.py` from [`file` project](https://github.com/file/file/blob/master/python/magic.py) without any modification. Debian package `python-magic` contains the binding from `file` project.